charts/datadog/values.yaml

```diff
@@ -1175,6 +1175,19 @@ clusterAgent:
   additionalLabels: {}
     # key: "value"
 
+  # clusterAgent.containerExclude -- Exclude containers from the Cluster Agent
```

Comment:
we can add the fact that this option will have a real effect only from cluster-agent 7.50.0+